data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +518,1020 @@ mission "Unfettered Jump Drive Trading"
+			`	When it is their turn in line, a couple towing four children with them, begin to plead. <firstgroup> The officials, perhaps the "Chiefs", which you have heard lead the Unfettered, speak among themselves. Then they turn back to the petitioners, and together, close their hand and raise it up, before lowering it, as if presenting something. By the relief on the couple's faces, you assume their request has been granted.`
+			choice
+				`	(Move a bit to see the next petitionners as they approach.)`
+			`	The next in line seems quite old, but he still is moving around proudly, as if not limited by his age at all. <secondgroup> This one, probably a soldier of some kind, clearly has some special status or a reputation which has earned him great respect for him to speak this way. Or maybe he just exemplifies what you've heard about the ways of the Unfettered; they bow to no one.`
+			`	The soldier moves away, and a tense silence hangs in the air for a moment. The Chiefs turn from the audience to speak among themselves, while mutterings and debate seem to be spreading throughout the crowd. This subject appears to be something for which strong opinions are held. The volume increases as the Chiefs continue their aside, and you begin to notice some of the nearby Hai turning their attention towards you, some even pointing you out. It may be that you are not welcome here. One of the younger Hai to notice you, probably still an adolescent judging by her colors, approaches you directly. She speaks softly, in your language, but clearly enough to be heard over the back and forth murmuring of the crowd. "Hi," she says with a warm, somewhat embarrassed chuckle, "I have noticed you easily among Hai. Do you want me to help translate what they say?"`
+			choice
+				`	"Yes, thank you."`
+					goto translate
+				`	"Thank you, but I already understand what I need to."`
+					goto polite
+				`	"I have no need of help."`
+			`	She seems a bit offended you do not wish her help.`
+			label polite
+			`	She does not press the matter further and proceeds to move away. Most of the Hai here are with what seems to be families or groups of friends, but you do not see your would-be helper join anyone else as she returns to her previous spot in the shifting audience.`
+			choice
+				`	(Ignore her.)`
+					goto headoff
+				`	(Go and speak with her.)`
+			`	You can see she's clearly happy you came to speak to her. You ask her why she seems alone, and she answers, looking at the ground with what seems to be contained sadness, "I have no family here. I lost my parents to a pirate raid, so I came here to find myself. I did not expect all of this, though. Did you change your mind about the translation?"`
```

Comment:
Because you're not, you're just as lost as she is
Because she needs smbd to speak to, as well

Reply:
But just because someone is lost doesn't mean you start telling your life story to them because you're lost too. She has no idea if you're likely to tell other people about this stuff. She doesn't know who you are or what you're like.

Reply:
so maybe put this in a later conversation, you mean?
`you ask her why she seems alone` thats putting words in the player's mouth kinda as well, and this mission is big enough

Reply:
Yes, I would have her tell you this when you know each other better.